nuxt.config.ts

```diff
@@ -4,7 +4,7 @@ import { compileTemplate } from 'vue/compiler-sfc'
 // https://nuxt.com/docs/api/configuration/nuxt-config
 export default defineNuxtConfig({
   srcDir: './src',
-  css: ['@/styles/main.css'],
+  css: ['@/assets/styles/main.css'],
```

Comment:
I moved styles to the initial Nuxt folder structure. Previous solution was a bit experimental